### PR TITLE
Add docstrings to `patroni.exceptions`

### PR DESCRIPTION
**Pull request by barthisrael**
References: PAT-119.

### CODE REVIEW

File: patroni/exceptions.py [hunk 1]
```diff
@@ -1,33 +1,55 @@
...
 
 
 class PostgresException(PatroniException):
+    """Any exception related with Postgres management."""
+
     pass
 
 
 class DCSError(PatroniException):
+    """Parent class for all kind of DCS related exceptions."""
```
Comment (hughcapet) [inline patroni/exceptions.py:35]: Is "*all* kind" actually correct here? @CyberDem0n

Comment (CyberDem0n) [reply patroni/exceptions.py:35]: I think yes. And they are handled in the Ha.

**Comment by CyberDem0n** [comment]
:+1:

**Comment by hughcapet** [comment]
👍

**Pull request merged**